Documentation/contributing/release/stable.rst

```diff
@@ -92,6 +91,10 @@ If you intent to release a new feature release, see the
             [...]
             <<end of add-to-NEWS.rst>>
 
+#. Generate the release notes by running the instructions provided in github.com/cilium/release
```

Comment:
This URL doesn't work, are you planning to push that repo before merging?

Reply:
No, but the repo will have that name ;-)